packages/perseus/src/server-item-renderer.tsx

```diff
@@ -149,6 +149,11 @@ export class ServerItemRenderer
     ) => {
         if (newFocus != null) {
             this._setCurrentFocus(newFocus);
+
```

Comment:
We need to call the interactionCallback in both handleInteractWithWidget and onFocus, as we're currently using it to close a feedbackPopover in webapp. I believe there are may be some future use cases where we want to know if a user is currently clicking into widgets.